README.rst

```diff
@@ -203,6 +203,14 @@ To use this source, you should first `apply <https://developer.oxforddictionarie
 .. image:: https://i.imgur.com/UO5nQjU.png
 
 
+`iTaigi-愛台語 <https://itaigi.tw/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: https://user-images.githubusercontent.com/1645228/55276638-07bf6f00-5331-11e9-990c-648595295e29.jpg
```

Comment:
There are some tofus in the screenshot. Is it possible to get rid of it?

Reply:
sure. Let me find another font.